README-koKR.md

```diff
@@ -531,10 +531,28 @@ render status: :forbidden
   end
   ```
 
-* <a name="beware-update-attribute"></a>
-  [`update_attribute`](http://api.rubyonrails.org/classes/ActiveRecord/Persistence.html#method-i-update_attribute) 메소드의 동작에 조심해야 한다.
-  (`update_attributes`와는 달리) 모델 validation를 실행하지 않기 때문에 모델의 상태가 손상되기 쉽다.
-<sup>[[link](#beware-update-attribute)]</sup>
+* <a name="beware-skip-model-validations"></a>
+  [다음](http://guides.rubyonrails.org/active_record_validations.html#skipping-validations)
+  메소드의 동작에 조심해야 한다. 이들은 모델 유효성 검증을 하지 않으므로,
+  간단하게 모델의 상태가 손상될 수 있다.
+<sup>[[link](#beware-skip-model-validations)]</sup>
+
+  ```Ruby
+  # 나쁜 예
+  Article.first.decrement!(:view_count)
+  DiscussionBoard.decrement_counter(:post_count, 5)
+  Article.first.increment!(:view_count)
+  DiscussionBoard.increment_counter(:post_count, 5)
+  person.toggle :active
+  product.touch
+  Billing.update_all("category = 'authorized', author = 'David'")
+  user.update_attribute(website: 'example.com')
```

Comment:
you're right OTL it's from upstream master, anyway I will fix it here and open pr to upstream.. ;_;